spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2521,6 +2525,33 @@ def body_pose_callback(self, data: Pose) -> None:
         mobility_params.body_control.CopyFrom(body_control)
         self.spot_wrapper.set_mobility_params(mobility_params)
 
+    def arm_joint_cmd_callback(self, data: JointState) -> None:
+        if not self.spot_wrapper:
+            self.get_logger().info(f"Mock mode, received arm joint commdn {data}")
+            return
+        arm_joint_map = {"sh0": None, "sh1": None, "el0": None, "el1": None, "wr0": None, "wr1": None}
+        # Check we have the right number of joints for the arm
+        if len(data.name) != len(arm_joint_map):
+            self.get_logger().warning(f"Expected {len(arm_joint_map)} joints, but received {len(data.name)}")
+            return
+
+        # Need to match the joint names in the JointState message to the joint names in the order we expect for spot.
+        # Depending on how the Spot is launched, the joint names could come in with a namespace or arm precusor such as
+        # `Spot/arm_sh0` or "arm_sh0" or simply just "sh0" 
+        for (name, position) in zip(data.name, data.position):
```

Comment:
```suggestion
        for name, position in zip(data.name, data.position):
```

Black wants it to be this^. That will fix your CI error